sdk/src/Core/AWSSDK.Core.NetStandard.csproj

```diff
@@ -23,6 +23,8 @@
     <NoWarn>$(NoWarn);CS1591;CA1822</NoWarn>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <SignAssembly>True</SignAssembly>
+
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
@danielmarbach can you add this same line to the `AWSSDK.Core.NetFramework.csproj` file? The internal builds are failing for the .NET Framework project because the PR doesn't allow unsafe code in .NET Framework.

Reply:
Sure. FYI all my PRs have push right